weed/s3api/chunked_reader_v4.go

```diff
@@ -333,12 +489,29 @@ func (cr *s3ChunkedReader) getChunkSignature(hashedChunk string) string {
 
 // readCRLF - check if reader only has '\r\n' CRLF character.
 // returns malformed encoding if it doesn't.
-func readCRLF(reader io.Reader) error {
+func readCRLF(reader *bufio.Reader) error {
```

Comment:
The correct type being passed in `*bufio.Reader` so I updated the signature, which allowed me to use `reader.Read` and `reader.Peek`. See the next comment to show where